wgpu-core/src/track/buffer.rs

```diff
@@ -327,8 +327,7 @@ impl<A: HalApi> ResourceTracker<BufferId, Buffer<A>> for BufferTracker<A> {
                 let existing_ref_count = self.metadata.get_ref_count_unchecked(index);
                 //2 ref count if only in Device Tracker and suspected resource itself and already released from user
                 //so not appearing in Registry
```

Comment:
this comment is super important and a bit cryptic to me. If I understand correctly "a resource is exactly then unused if it is solely owned by the device tracker itself and by the caller (originating from the suspected to be destroyed list), i.e. up to two strong references"?

Also, the method comment still talks about three references..?

Reply:
I read it and I agree - it's not really clear :)
Would be better something like:
"RefCount 2 means that resource is hold just by DeviceTracker and this suspected resource itself - so it's already been released from user and so it's not inside Registry\Storage"

Reply:
I like this version better.